src/core/data.jl

```diff
@@ -48,7 +48,7 @@ function count_active_items(network::Dict{String, Any})
         active_count["nw"] = Dict{String,Any}(nw => sum(length(comp_ids) for (comp_type,comp_ids) in repariable_network) for (nw, repariable_network) in active_set["nw"] )
     else
         active_set = get_active_items(network)
-        return active_count = sum(length(comp_ids) for (comp_type,comp_ids) in active_set)
+        active_count = sum(length(comp_ids) for (comp_type,comp_ids) in active_set)
     end
     return active_count
 end
```

Comment:
`count/get_active items` can be moved to tests/common. I was previously using it outside of the package to query results, but it doesn't make sense to keep it in anymore.

I think its fair to make the `count_*_items` functions only usable on single networks, and have a return type of Int. Again, the only use for querying muti-networks is for verifying solutions.